app/controllers/webapp/Auth.scala

```diff
@@ -347,7 +373,7 @@ class Auth @Inject() (
   def handleResetPassword(tokenId:String) = Action.async(parse.json) { implicit request =>
     val data = request.body.validate[Password]
     data.fold(
-      errors => Future.successful(WebAppResult.Bogus(request, "error.bogusResetPasswordData", Nil, "AuthProvider.HandleResetPassword.BogusData", JsError.toJson(errors)).getResult),
+      errors => Future.successful(WebAppResult.Bogus(request, "error.bogusResetPasswordlData", Nil, "AuthProvider.HandleResetPassword.BogusData", JsError.toJson(errors)).getResult),
```

Comment:
Why did you added `error.bogusResetPasswordlData`? It was `error.bogusResetPasswordData` before (without the `1`). So did you've added a new message string?

Reply:
@deinelieblings you can simply accept my suggestion here on GitHub, if the `1` is a mistake. No need to check it out locally...